packages/google_sign_in/google_sign_in_ios/darwin/google_sign_in_ios/Package.swift

```diff
@@ -0,0 +1,46 @@
+// swift-tools-version: 5.9
+
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import PackageDescription
+
+let package = Package(
+  name: "google_sign_in_ios",
+  platforms: [
+    .iOS("12.0"),
+    .macOS("10.15")
```

Comment:
This is bumped up to macOS 10.15. Required to depend on [GoogleSignIn](https://github.com/google/GoogleSignIn-iOS/blob/65fb3f1aa6ffbfdc79c4e22178a55cd91561f5e9/Package.swift#L26)

Reply:
We [already require 10.15](https://github.com/flutter/packages/blob/4b00343963a1529598e602f0e3954c9fee9ec7d5/packages/google_sign_in/google_sign_in_ios/darwin/google_sign_in_ios.podspec#L28) (which is good, because otherwise this would be very hard to roll out in a non-disruptive way).